tests/integration_tests/security/test_seccomp.py

```diff
@@ -170,3 +170,35 @@ def test_advanced_seccomp_malicious(tmp_advanced_seccomp_binaries):
 
     # The demo malicious firecracker should have received `SIGSYS`.
     assert outcome.returncode != 0
+
+
+def test_seccomp_applies_to_all_threads(test_microvm_with_api):
+    """Test all Firecracker threads get default seccomp level 2"""
+    test_microvm = test_microvm_with_api
+    test_microvm.spawn()
+
+    # Set up the microVM with 2 vCPUs, 256 MiB of RAM and
+    # a root file system with the rw permission.
+    test_microvm.basic_config()
+
+    test_microvm.start()
+
+    # Get Firecracker PID so we can count the number of threads.
+    cmd = 'lsof -U | grep {} | awk \'{{print $2}}\''.format(
```

Comment:
The integration tests run with the jailer, so you should have the pid already in `test_microvm._jailer_clone_pid`

Reply:
Thanks, updated!